### DISCUSSION TITLE
Drawing Extended Hours?

### DISCUSSION THREAD

**Discussion by netshade** [tradingview/lightweight-charts#779] [Q&A] [answered]
I want to draw a custom background for extended hours trading on lightweight chart. The only way I can currently think to do that is to add an Area series for the different extended hours periods. Is there a better way to accomplish this?

**Comment by timocov** [comment, marked as answer]
> Is there a better way to accomplish this?

Don't think so. Probably #408 might be an option, but it is not implemented yet.

**Reply by netshade** [reply]
Cool, thanks for link
